### DISCUSSION TITLE
how install this plugin and add csv  in docker ?

### DISCUSSION THREAD

**Discussion by rodrigola** [grafana/grafana-csv-datasource#137] [Q&A]
hello i have linux and instaled  grafana in docker 
so how intall this plugin in grafana 

i want open csv in my linux in the folder hassio/share

i fond some exemples : 

docker run -d \
     -p 3000:3000 \
      --name=grafana \
    -e "GF_SERVER_ROOT_URL=http://grafana.noto.local" \
    -e "GF_SECURITY_ADMIN_PASSWORD=noto" \
    -e "GF_INSTALL_PLUGINS=alexanderzobnin-zabbix-app" \
    -e "GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=alexanderzobnin-zabbix-datasource" \
    --restart unless-stopped \
    -v grafana-storage:/var/lib/grafana \
       grafana/grafana:7.2.0-ubuntu

i think to install to open a csv in my folder i only need to add 
this line 
    -e "GF_INSTALL_PLUGINS=marcusolsson-csv-datasource" \

correct ?

this is the exemple of csv i want open

Volt,Volt,vento,luz,som,ar,time
--
,,,,,,
127.167,127.091,0.06,-0.118,0,0,"18/11/21,   2:25:01, ( Thursday )"
127.167,127.091,0.06,-0.118,0,0,"18/11/21,   2:25:04, ( Thursday )"
127.167,127.091,0.06,-0.118,0,0,"18/11/21,   2:25:06, ( Thursday )"
127.167,127.091,0.06,-0.118,0,0,"18/11/21,   2:25:10, ( Thursday )"
127.167,127.091,0.06,-0.118,0,0,"18/11/21,   2:25:12, ( Thursday )"
127.167,127.091,0.06,-0.118,0,0,"18/11/21,   2:25:16, ( Thursday )"
127.1,127.091,0.06,-0.118,0,0,"18/11/21,   2:25:19, ( Thursday )"
127.1,127.211,0.007,-0.065,0,0,"18/11/21,   2:25:21, ( Thursday )"
127.167,127.211,-0.087,-0.065,0,0,"18/11/21,   2:25:25, ( Thursday )"
127.167,127.308,-0.035,-0.118,0,0,"18/11/21,   2:25:27, ( Thursday )"
127.123,127.308,-0.111,-0.118,0,0,"18/11/21,   2:25:30, ( Thursday )"
127.123,127.043,-0.146,-0.083,0,0,"18/11/21,   2:25:34, ( Thursday )"

**Comment by marcusolsson** [comment]
Local mode is disabled by default (for security reasons), so you need to [enable it](https://marcus.se.net/grafana-csv-datasource/configuration#allow-local-mode).

**Comment by rodrigola** [comment]
[plugin.marcusolsson-csv-datasource]
allow_local_mode = true 
how i do this ? i fond in reddit this 

GF_INSTALL_PLUGINS=alexanderzobnin-zabbix-app,grafana-clock-panel,marcusolsson-csv-datasource
GF_PLUGIN_MARCUSOLSSON_CSV_DATASOURCE_ALLOW_LOCAL_MODE=true

**Reply by rodrigola** [reply]
/var/lib/docker/volumes/grafana-storage/_data/plugins/marcusolsson-csv-datasource# dir
CHANGELOG.md                          LICENSE
gpx_csv-datasource_darwin_amd64       MANIFEST.txt
gpx_csv-datasource_darwin_arm64       module.js
gpx_csv-datasource_linux_amd64        module.js.LICENSE.txt
gpx_csv-datasource_linux_arm          module.js.map
gpx_csv-datasource_linux_arm64        plugin.json
gpx_csv-datasource_windows_amd64.exe  README.md
img
 ????

**Reply by marcusolsson** [reply]
If you want to configure it using environment variables, you might need to use: 

```
GF_PLUGIN_ALLOW_LOCAL_MODE=true
```
